src/bindings/python/tests/test_graph/test_constant.py

```diff
@@ -389,6 +389,7 @@ def test_float_to_f8e8m0_constant_matrix(ov_type, numpy_dtype, opset):
 
     compressed_const = opset.constant(data, dtype=ov_type, name="fx_constant")
     convert_to_fp8 = opset.convert(compressed_const, Type.f8e8m0)
+    convert_to_fp8.get_rt_info()["DisableEliminateUselessConvert"] = "1"
     convert_back = opset.convert(convert_to_fp8, ov_type)
     parameter = opset.parameter(ov.PartialShape([-1, -1]), ov_type)
```

Comment:
@maxnick Because I remove useless convert(a->b, b->a), but this test still test it. so I just add rt_info to disable it.

Reply:
Reverted.